examples/Replay

```diff
@@ -0,0 +1,46 @@
+#include <Servo.h>
+#include <EEPROM.h>
+
+Servo myServo;        // create servo object to control a servo
+
+bool record = false;  // boolean to determine if recording is taking plce
+int potpin = 0;       // analog pin used to connect the potentiometer
+int val;              // variable to read the value from the analog pin
+int read;             // variable to read the data from memory
+
+void setup() {
+  pinMode(13, OUTPUT); // button on pin 2
```

Comment:
```suggestion
  pinMode(buttonPin, INPUT_PULLUP); // configure the button's pin
```
- You used a variable to define the potentiometer's pin, so to be consistent use a variable to define the button's pin as well. I'll leave it to you to add the declaration of `buttonPin`.
- A button pin must be in input mode, not output mode!
- The pin number used in the code doesn't match the one in the comment